include/sdf/Element.hh

```diff
@@ -112,6 +112,14 @@ namespace sdf
     /// \sa Element::SetRequired
     public: const std::string &GetRequired() const;
 
+    /// \brief Set if the element was set or default
+    /// \param[in] _value True if the element is set
+    public: void SetSet(const bool _value);
```

Comment:
Can we use a more descriptive function? eg. `SetIsExplicitlySet` or `SetIsSetByUser`. My examples aren't great either, but I don't have better suggestions :)